extlinks/templates/400.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
```

Comment:
You need to add the templates to the `urls.py` file. See the TWLight example here: https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/urls.py